omero/institution/getting-started.html

```diff
@@ -9,8 +9,108 @@
         <hr class="whitespace">
         <div class="row column text-center">
             <h3 class="header-green">Getting Started</h3>
-            <p class="subheader">Whether you’re an experienced system administrator ready to jump straight into installing OMERO.server or a manager wondering if the OMERO platform will meet your facility’s needs, the following guides will get you started.</p>
+            <p class="subheader">Whether you’re an experienced system administrator ready to jump straight into installing OMERO.server or a manager wondering if the OMERO platform will meet your facility’s needs, this page will get you started.</p>
         </div>
+        <!-- begin Benefits -->
+        <hr class="whitespace">
+        <div class="row column text-center">
+            <h3>What do I get when I install OMERO?</h3>
+        </div>
+        <hr>
+        <div class="row">
+            <div class="medium">
+                <p>A complete platform for managing your images, <a href="{{ site.baseurl }}/omero/" target="_blank">OMERO</a> has many <a href="{{ site.baseurl }}/omero/features" target="_blank">features</a> and <a href="{{ site.baseurl }}/omero/developers/" target="_blank">expansion points</a> to <a href="{{ site.baseurl }}/omero/features/analyze/" target="_blank"> integrate with your existing tools</a>. OMERO is <a href="{{ site.baseurl }}/omero/scientists/" target="_blank"> designed with scientists in mind</a> and is tried and tested in hundreds of institutions worldwide. In this section, you might get an idea what are the benefits of OMERO for your institution. 
+                </p>
+                <p>The software is for free, but do not miss the <a href="#what-is-needed">What is needed to host OMERO?</a> section below. Also [LINK TO NOTE ABOUT SUPPORT LEVELS].</p>
```

Comment:
Maybe "The software can be downloaded for free" or "is available for free"

Reply:
LINK TO NOTE ABOUT SUPPORT LEVELS - still to be filled in

Reply:
fixed in https://github.com/ome/www.openmicroscopy.org/pull/462/commits/b34402411fec182b880b40934783bd667f93945f

for the missing link, made a ToDo list in the header of the PR